systests/driver/media_driver.go

```diff
@@ -44,7 +44,7 @@ const aeronClientLivenessTimeoutNs = "aeron.client.liveness.timeout"
 // Must match Java's `PUBLICATION_UNBLOCK_TIMEOUT_PROP_NAME`, measured in nanos
```

Comment:
Move https://github.com/aayushduwadi/aeron-go/commit/22aa8ba0431b98ded35a57b66f494ab8d14a2529 into this PR